examples/resources/awscc_config_config_rule/config_config_rule.tf

```diff
@@ -0,0 +1,37 @@
+resource "awscc_config_config_rule" "r" {
```

Comment:
I would avoid ambiguous resource logical id such as `r` , can you please look at other example, at minimum you can use something like `example`